lib/src/command/dependency_services.dart

```diff
@@ -606,6 +606,24 @@ Map<String, PackageRange>? _dependencySetOfPackage(
           : null;
 }
 
+VersionConstraint _bumpConstraint(
```

Comment:
Could you add a small dartdoc explaining
```suggestion
/// Return a version compatible with [newVersion].
///
/// By convention if the dependency is pinned in [originalVersion] we bump it to a pinned version. Otherwise use [VersionConstraint.compatibleWith].
VersionConstraint _bumpConstraint(
```